### ISSUE TITLE
[MaUI-Windows] Touch scroll is not working in a scroll view when adding pinch gesture to the child.

### ISSUE THREAD

**Issue by Deepak1799** [dotnet/maui#5612]
### Description

When adding pinch gesture recogniser to child (content view) of a scroll view. Touch scroll is not working. It looks the pinch gesture blocks the fling gesture of scroll view. 

### Steps to Reproduce

1. Create File -> New -> MaUI App.
2. Add a scroll view control.
3. Add a content (image or some Views) to scroll view and perform touch scroll. **It work as expected**
4. Add pinch gesture recognizer to the content view as shown in the example: https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/gestures/pinch 
5. Now perform the touch scroll. **It will not work**

### Version with bug

Preview 14 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

Windows SDK 10.0.19041

**Comment by v-longmin** [comment]
@Deepak1799 Could you share your repro project for us so that we can reproduce this issue, thanks?

**Comment by Deepak1799** [comment]
@v-longmin , Thank you following up on the reported problem. I have attached the repro project below:
[PinchToZoomV1.zip](https://github.com/dotnet/maui/files/8368262/PinchToZoomV1.zip)

**Comment by VincentBu** [comment]
Hi @Deepak1799 , are you trying to deploy the app on a tablet device? I add some labels under your code and I can scroll by mouse.
![image](https://user-images.githubusercontent.com/44959937/160549141-4e3d0397-a082-4bdb-b787-b3497916f6c3.png)

**Comment by Deepak1799** [comment]
@VincentBu , No, I have used desktop application and it looks the issue inconsistently occurred. However, one of the issues (either touch scroll or pinch is not working) is consistently occur at a time. When it scroll, but pinch to zoom is not working. Can you please try pinch to zoom if you can scroll the contents?

**Comment by Deepak1799** [comment]
@VincentBu , No, I have used desktop application. I have made changes (placed the content view inside the scroll view) to repro the touch scroll issue consistently. Can you please try this modified sample?

[PinchToZoomV1.zip](https://github.com/dotnet/maui/files/8371900/PinchToZoomV1.zip)

**Note:** Mouse scroll will work, but touch scroll will not work. When removing the line of code "GestureRecognizers.Add(pinchGesture);", the touch scroll will work properly.

**Comment by VincentBu** [comment]
repro on mac m1. as [Deepak1799](https://github.com/Deepak1799) said, mouse scroll works while touch scroll doesn't.

**Comment by flipper09112** [comment]
updates?

**Comment by Zhanglirong-Winnie** [comment]
Verified this issue with Visual Studio Enterprise 17.7.0 Preview 5.0. Not repro on android platform with sample project.Mouse scroll and touch scroll will work.
[PinchToZoomV1.zip](https://github.com/dotnet/maui/files/12288146/PinchToZoomV1.zip)
